### DISCUSSION TITLE
is project still maintained ?

### DISCUSSION THREAD

**Discussion by pada57** [atifaziz/NCrontab#101] [Q&A] [answered]
Hello

as i don't see any commits since  a year and last release even older, is it still a maintained project ? or is there any news one forking that one ?

**Comment by atifaziz** [comment, marked as answer]
Yes, it is still maintained.

**Reply by pada57** [reply]
good to hear ! is there a release plan ?

**Reply by atifaziz** [reply]
Not for the immediate future as there are no high priority features in the queue. This is a simple library that is intended to do one thing and has been doing it for some years without too many bells and whistles. _Maintained_ therefore means high priority bugs will be fixed, and sooner than later if a quality patch is contributed.
